vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -215,13 +215,20 @@ class LedgerImpl : public ledger::Ledger,
       const ledger::UrlMethod method,
       ledger::LoadURLCallback callback);
 
-  virtual void ReconcileComplete(
+  // DEPRECATED
```

Comment:
Question/Curious - Is there a cross platform preprocessor macro for deprecating methods? I've got one (of my own) for Windows & Mac, but not sure if it would work on Linux/Android/iOS even. Would it be useful to investigate at some point?

Reply:
not sure, I just added them as I will be removing them in 1.6